packages/material-ui-lab/src/useAutocomplete/useAutocomplete.js

```diff
@@ -263,7 +264,7 @@ export default function useAutocomplete(props) {
       )
     : [];
 
-  popupOpen = freeSolo && filteredOptions.length === 0 ? false : popupOpen;
+  popupOpen = freeSolo && filteredOptions.length === 0 && !loading ? false : popupOpen;
```

Comment:
I don't think that my proposed solution was the best possible. Especially, I don't think that we should leak the `loading` concerns down to the `useAutocomplete` hook. I have an alternative proposal.